### PR TITLE
Maintainance

### PR DESCRIPTION
**Pull request by ksol**
@Soulou I was notified of https://github.com/Scalingo/omniauth-scalingo/pull/3 and noticed the failing CI. This PR should restore everything. I'm leaving the compat with 2.7 (end of life) since we still need it for a little while.

I had to fix the specs too, not just the linting, but I'm not entirely confident I've done it right.

### CODE REVIEW

File: lib/omniauth/strategies/scalingo.rb [hunk 1]
```diff
@@ -23,32 +23,32 @@ def authorize_params
...
       end
 
       def raw_info
-        @raw_info ||= access_token.get('/v1/users/self').parsed['user']
+        @raw_info ||= access_token.get("/v1/users/self").parsed["user"]
       end
 
       def callback_url
-        full_host + script_name + callback_path
+        full_host + callback_path
```
Comment (Soulou) [inline lib/omniauth/strategies/scalingo.rb:48]: I see this change here, what's the impact?

Comment (ksol) [reply lib/omniauth/strategies/scalingo.rb:48]: omniauth 2+ respects script_name, cf the bottom of the release notes : https://github.com/omniauth/omniauth/releases/tag/v2.0.0

**Review by Soulou — APPROVED** [review]
LGTM then

**Comment by ksol** [comment]
I've added a changelog and bump the version of the lib from 2.0 to 2.1.

**Pull request merged**